Samples/iOS-Swift/iOS-Swift/SentrySDKWrapper.swift

```diff
@@ -234,6 +233,7 @@ extension SentrySDKWrapper {
         }
     }
     
+    // swiftlint:disable cyclomatic_complexity
```

Comment:
`m`: Instead of enabling and disabling the linting, please consider refactoring the method into smaller ones

Reply:
It's ok, I moved the inner `removeHook` function out, and the parallel logic into a new `createHookFile` function, so that makes the linter happy. I'd rather do that than write justification for overruling the linter.

Reply:
I'm ok to change this, I moved the inner `removeHook` function out, and extracted a parallel `createHook` function, renaming the big one to `updateHook`, and that makes the linter happy. Would rather just do that than write justifications in comments.